src/audio/winamp/winamp.cpp

```diff
@@ -73,14 +73,32 @@ namespace SoLoud
 
 		mOffset = 0;
 		mStreamPosition = 0.0f;
-		return 0;
+		return SO_NO_ERROR;
+	}
+
+	result WinampInstance::seek(double aSeconds, float* mScratch, unsigned int mScratchSize)
+	{
+		mParent->inPlugin->setOutputTime(aSeconds);
+		mStreamPosition = aSeconds;
+		return SO_NO_ERROR;
 	}
 
 	bool WinampInstance::hasEnded()
 	{
 		return !(mFlags & AudioSourceInstance::LOOPING) && mParent->outPlugin->finishedPlaying();
 	}
 
+	float WinampInstance::getInfo(unsigned int aInfoKey)
+	{
+		// Can resume music
+		if (aInfoKey == 7777) {
+			// Needs to Duplicate
+			return 1.0f;
+		}
+
+		return 0.0f;
+	}
+
```

Comment:
This section needs to be added to VGMStream too I guess, as the whole flow of "can I resume" is called in `music.cpp` therefore, also VGMStream is impacted by this change. Would you be able to add the required logic over there too?

Reply:
Yes I can do that
Just know that vgmstream is currently not impacted. getInfo returns 0.0 in AudioSourceInstance parent implementation.

Reply:
Ah ok so if there's no impact then it's not needed :) Thanks! Fine by you if I merge this then?

Reply:
No don't merge it, I have more changes to push :)